Src/PointStream.inl

```diff
@@ -153,7 +153,7 @@ void PLYInputPointStream< Real , Dim >::reset( void )
 			exit( 0 );
 		}	
 
-		if( elem_name=="vertex" , elem_name )
+		if( elem_name=="vertex" )
```

Comment:
I'm not 100% certain this change preserves the behavior of the original, but I couldn't figure out the reason to compare `elem_name=="vertex"` and disregard that result, then try to cast `elem_name` to a boolean which caused a compiler error. I may be misinterpreting, but this looked just a little fishy.